soh/src/code/z_player_lib.c

```diff
@@ -650,6 +651,7 @@ void Player_SetModels(Player* this, s32 modelGroup) {
     this->waistDLists = &sPlayerDListGroups[gPlayerModelTypes[modelGroup][4]][gSaveContext.linkAge];
 
     Player_SetModelsForHoldingShield(this);
+    GameInteractor_ExecuteOnPlayerChangeItem();
```

Comment:
Not necessarily only/always called when changing items, this would be a more appropriate name
```suggestion
    GameInteractor_ExecuteOnPlayerSetModels(modelGroup);
```